packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -279,6 +281,10 @@ export default class Meetings extends WebexPlugin {
       this.webex.internal.mercury.on(ROAP.ROAP_MERCURY, (envelope) => {
         MeetingsUtil.handleRoapMercury(envelope, this.meetingCollection);
       });
+
+      this.webex.internal.mercury.on(ONLINE, () => {
```

Comment:
Should this be a `once` or do we want to do this every time mercury emits online?

Reply:
when ever mercury connects and disconnects we need to do that

Reply:
Do we clean up or turn off the events on disconnect?